platform/linux-generic/include/odp/api/plat/atomic_inlines.h

```diff
@@ -161,9 +161,9 @@ _ODP_INLINE void odp_atomic_min_u32(odp_atomic_u32_t *atom, uint32_t val)
 
 /**
  * @internal
- * CAS operation expression for the ATOMIC_OP macro
+ * CAS operation expression for the ODP_ATOMIC_OP macro
```

Comment:
Better to use `_ODP_` prefix since this is not an official API.

Reply:
v2